src/main/kotlin/no/nav/syfo/api/endpoints/SenOppfolgingEndpoints.kt

```diff
@@ -0,0 +1,55 @@
+package no.nav.syfo.api.endpoints
+
+import io.ktor.http.*
+import io.ktor.server.application.*
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+import no.nav.syfo.api.model.toResponseDTO
+import no.nav.syfo.application.SenOppfolgingService
+import no.nav.syfo.infrastructure.NAV_PERSONIDENT_HEADER
+import no.nav.syfo.infrastructure.clients.veiledertilgang.VeilederTilgangskontrollClient
+import no.nav.syfo.infrastructure.clients.veiledertilgang.validateVeilederAccess
+import no.nav.syfo.util.getNAVIdent
+import no.nav.syfo.util.getPersonident
+import java.util.*
+
+const val kandidatUuidParam = "vedtakUUID"
+const val senOppfolgingApiBasePath = "/api/internad/v1/senoppfolging"
+const val ferdigbehandlingPath = "/kandidat/{$kandidatUuidParam}/ferdigbehandling"
```

Comment:
Hvis vi ønsker å følge REST konvensjoner så skal ressursgrupper (i dette tilfellet `/kandidat`) skrives som flertall. REST verbet bør også være det eneste "verbet" i url'en og beskriver hva som skal gjøres med ressursen. Så kan vi bruke `PUT`/`POST`/`PATCH` for å vise at det gjøres en endring 😊 Synes å følge REST generelt gjør API'er mer oversiktlige og lettere å skjønne hva som skjer.
```suggestion
const val ferdigbehandlingPath = "/kandidater/{$kandidatUuidParam}"
```

Reply:
Her kunne man til og med utvidet enda mer om vi vil være enda mer spesifikke mtp. hvilken ressurs vi egentlig jobber på:
POST `/kandidater/{$kandidatUuidParam}/vurderinger`
Da hadde det vært enda tydeligere at det man gjør er å lage en ny vurdering på en eksisterende kandidat.

Reply:
Ja, lurte på å bruke `/vurderinger` i stedet, men per nå er det bare ferdigbehandle vi gjør her (hardkodet status). 
Kunne jo sendt inn statusen i en vurdering-body fra klienten i stedet, men usikker på hvor generell vi skal gjøre den uten at vi vet om det kommer flere typer vurderinger :thinking:

Reply:
Mtp diskusjonen på standup om at vi antagelig kommer til å lagre noe mer info om vurderingen etter hvert så er det kanskje like greit å endre til `/vurderinger` og ta i mot en body med `status: FERDIGBEHANDLET` som kan utvides senere. Høres det ok ut @vetlesolgaard ?

Reply:
Tenkte det samme på standup. Vi sender helt sikkert med noe metadata her etterhvert. Høres veldig bra ut det foreslår! 😊

Reply:
Refaktorert nå. Blir en egen type enum på vurdering som bare kan ha verdien `FERDIGBEHANDLET` per nå.